solr/bin/solr.cmd

```diff
@@ -314,7 +313,7 @@ goto done
 @echo   -f            Start Solr in foreground; default starts Solr in the background
 @echo                   and sends stdout / stderr to solr-PORT-console.log
 @echo.
-@echo   -c or -cloud  Start Solr in SolrCloud mode; if -z not supplied and ZK_HOST not defined in
+@echo   -c or --cloud Start Solr in SolrCloud mode; if -z not supplied and ZK_HOST not defined in
```

Comment:
correct!   `--cloud` is what the docs should say.  We need to preserve the use of `-cloud` in 9x however...